### ISSUE TITLE
`mint.close()` not working

### ISSUE THREAD

**Issue by dzg** [mrooney/mintapi#215]
Recently I started getting this when doing `mint.close()`:

`selenium.common.exceptions.WebDriverException: Message: unknown error: Element <a id="link-logout" class="navbar-link" href="/logout.event?task=explicit&amp;country=">...</a> is not clickable at point (704, 32). Other element would receive the click: <div data-automation-id="ModalDialog--container" open="" role="dialog" class="StyledModalDialog__Container-cn89yw-1 iIlfZT" style="z-index: 102;">...</div>`

Full traceback:
```
Traceback (most recent call last):
  File "<stdin>", line 1, in <module>
  File "/usr/local/lib/python3.7/site-packages/mintapi/api.py", line 400, in close
    self.driver.find_element_by_id('link-logout').click()
  File "/usr/local/lib/python3.7/site-packages/selenium/webdriver/remote/webelement.py", line 80, in click
    self._execute(Command.CLICK_ELEMENT)
  File "/usr/local/lib/python3.7/site-packages/selenium/webdriver/remote/webelement.py", line 633, in _execute
    return self._parent.execute(command, params)
  File "/usr/local/lib/python3.7/site-packages/selenium/webdriver/remote/webdriver.py", line 321, in execute
    self.error_handler.check_response(response)
  File "/usr/local/lib/python3.7/site-packages/selenium/webdriver/remote/errorhandler.py", line 242, in check_response
    raise exception_class(message, screen, stacktrace)
selenium.common.exceptions.WebDriverException: Message: unknown error: Element <a id="link-logout" class="navbar-link" href="/logout.event?task=explicit&amp;country=">...</a> is not clickable at point (704, 32). Other element would receive the click: <div data-automation-id="ModalDialog--container" open="" role="dialog" class="StyledModalDialog__Container-cn89yw-1 iIlfZT" style="z-index: 102;">...</div>
  (Session info: headless chrome=83.0.4103.61)
  (Driver info: chromedriver=2.41.578706 (5f725d1b4f0a4acbf5259df887244095596231db),platform=Mac OS X 10.14.6 x86_64)
```

**Comment by robwil** [comment]
I believe this happens because Mint is now launching this modal on login:

![image](https://user-images.githubusercontent.com/605546/83952575-b699ed80-a807-11ea-870b-431eef8052f3.png)

Probably need to detect the modal and explicitly close it before trying to initiate the logout, but haven't had time to try that yet.

**Comment by jaron-l** [comment]
I'm having a very similar issue except the final exception on my stacktrace is:
`selenium.common.exceptions.ElementClickInterceptedException: Message: element click intercepted: Element <a id="link-logout" class="navbar-link" href="/logout.event?task=explicit&amp;country=">...</a> is not clickable at point (705, 32). Other element would receive the click: <div data-automation-id="ModalDialog--container" open="" role="dialog" class="StyledModalDialog__Container-cn89yw-1 iIlfZT" style="z-index: 102;">...</div>
  (Session info: headless chrome=81.0.4044.138)`

It seems odd that the pop up would only cause errors on logout if what @robwil says is correct.

**Comment by mrooney** [comment]
Should be fixed in master after merging #221, thanks for the report!
